h4/h4.ino

```diff
@@ -285,6 +286,22 @@ int savedStarts = 0; // starts saved in Preferences
 int nextStarts = starts; // number of starts that should be used asap
 bool nextStartsFlag = false; // whether nextStarts requires attention
 
+struct ToolOffset {
+    float xOffset;
+    float zOffset;
+};
+
+// Tool offsets in deci-microns (DU). 1mm = 10000 DU.
+ToolOffset toolOffsets[4] = {
+    {0, 0}, //primary tool always 0,0
+    {30000, 20000}, //secondary tool 3mm X, 2mm Z
+    {0, 0},
+    {0, 0}
+};
+
+ToolOffset currentOffset = {0, 0};
+int currentTool = 0;
```

Comment:
It would be consistent with the vars above to name these `toolOffset` and `tool`